packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1343,4 +1346,54 @@ void main() {
     expect(box.size.height, equals(128.0)); // _kPanelHeaderCollapsedHeight + 80.0 (double padding)
     expect(box.size.width, equals(736.0));
   });
+
+  testWidgets('ExpansionPanelList respect dividerColor', (WidgetTester tester) async {
+    const Color dividerColor = Colors.red;
+    await tester.pumpWidget(const MaterialApp(
+      home: SingleChildScrollView(
+        child: SimpleExpansionPanelListTestWidget(
+          dividerColor: dividerColor,
+        ),
+      ),
+    ));
+
+    final DecoratedBox decoratedBox = tester.widget(find.byType(DecoratedBox).last);
+    final BoxDecoration decoration = decoratedBox.decoration as BoxDecoration;
+
+    // For the last DecoratedBox, we will have a Border.top.
+    expect(decoration.border.top.color, dividerColor);
+  });
+
+  testWidgets('ExpansionPanelList.Radio respects DividerColor', (WidgetTester tester) async {
+    const Color dividerColor = Colors.red;
+    await tester.pumpWidget(MaterialApp(
+      home: SingleChildScrollView(
+        child: ExpansionPanelList.radio(
+          dividerColor: dividerColor,
+          children: <ExpansionPanelRadio>[
+            ExpansionPanelRadio(
+              headerBuilder: (BuildContext context, bool isExpanded) {
+                return Text(isExpanded ? 'B' : 'A', key: const Key('firstKey'));
+              },
+              body: const SizedBox(height: 100.0),
+              value: 0,
+            ),
+            ExpansionPanelRadio(
+              headerBuilder: (BuildContext context, bool isExpanded) {
+                return Text(isExpanded ? 'D' : 'C', key: const Key('secondKey'));
+              },
+              body: const SizedBox(height: 100.0),
+              value: 1,
+            ),
+          ],
+        ),
+      )
+    ));
+
+    final DecoratedBox decoratedBox = tester.widget(find.byType(DecoratedBox).last);
+    final BoxDecoration boxDecoration = decoratedBox.decoration as BoxDecoration;
+
+    // For the last DecoratedBox, we will have a Border.top.
```

Comment:
nit: slightly more description
```suggestion
    // For the last DecoratedBox, we will have a Border.top with the provided dividerColor.
```